mne/commands/mne_freeview_bem_surfaces.py

```diff
@@ -88,6 +88,10 @@ def run():
     subjects_dir = options.subjects_dir
     method = options.method
 
+    if (subjects_dir is None or subject is None or 
```

Comment:
`subjects_dir` can be None in regular use, as it can be pulled by `freeview_bem_surfaces` from the env or MNE config file via the line above:
```
subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
```
This should thus probably only check `subject`, and the file check should live in `freeview_bem_surfaces`. So it's better to move both checks there.

Reply:
Good points, thanks. Fixed.